### ISSUE TITLE
Offline Forceload on Server

### ISSUE THREAD

**Issue by Tomsauy** [thexaero/open-parties-and-claims#228]
I try to use forceload with offline option (activate in configuration file)  but with no effect
Forceload work if I am on the server but not if I leave 

Configuration file updated :  openpartiesandclaims-server-claim-config.toml
```
[playerConfig.claims.forceload]
	#When enabled, the chunks you have marked for forceloading are forceloaded.
	#If the forceload limit has changed and you have more chunks marked than the new limit, then some of the chunks won't be forceloaded. Unmark any chunks until you are within the limit to ensure that all marked chunks are forceloaded.
	enabled = true
	#When enabled, the chunks you have marked for forceloading stay loaded even when you are offline (can significantly affect server performance!).
	#If your forceload limit is affected by your FTB Ranks rank/permissions, then you need to login at least once after a server (re)launch for it to take effect while you are offline.
	offlineForceload = true
```

Game & Mods informations :

Minecraft: 1.19.2
Forge: 43.2.4
[Open Parties and Claims](https://www.curseforge.com/minecraft/mc-mods/open-parties-and-claims/files/4344793): 0.17.0
[Xaero's Minimap](https://www.curseforge.com/minecraft/mc-mods/xaeros-minimap/files/4418097): 23.3.2
[Xaero's World Map](https://www.curseforge.com/minecraft/mc-mods/xaeros-world-map/files/4424632): 1.29.4
[Storage Drawer](https://www.curseforge.com/minecraft/mc-mods/storage-drawers/files/3884263): 11.1.2
[Create](https://www.curseforge.com/minecraft/mc-mods/create/files/4371809): 0.5.0i

And some other mods

Is there any other configuration needed for this option to work ?

Thanks for your great job !

**Comment by thexaero** [comment]
You're editing the server claim config file which has nothing to do with your player character. Edit player configs through the user interface, not the files.

**Comment by thexaero** [comment]
In the main config file, you can add the offlineForceload option to player-configurable options so that each player can toggle it for themselves. Otherwise, change it in the default config (though the user interface, not the file).

**Comment by Tomsauy** [comment]
> You're editing the server claim config file which has nothing to do with your player character. Edit player configs through the user interface, not the files.

Sorry if I make a mistake, but "My Player Config" throught interface is disable for this option (grey/black)
So I edit "Default Player Config" throught interface it's ok but for all players not just for me
Is there any way to config offline forceload just for me ?

Sorry if I am not clear or my question are trivial 

Thanks

**Comment by thexaero** [comment]
While the server isn't running, edit _openpartiesandclaims-server.toml_, which is the main config file. There are options called _playerConfigurablePlayerConfigOptions_ and _opConfigurablePlayerConfigOptions_. The comments above the 2 options explain everything in detail. Look at how options are currently added there and do the same for `claims.forceload.offlineForceload`.

**Comment by thexaero** [comment]
I actually have the question about greyed out options in the FAQ: https://www.curseforge.com/minecraft/mc-mods/open-parties-and-claims

**Comment by Tomsauy** [comment]
Thanks a lot for all this precision I will double check that 
One more time thanks for what you do !!

**Comment by thexaero** [comment]
You're welcome!

**Comment by JayAitch** [comment]
Hey so I have a simular issue. Offline forceloading is disabled, and isnt modifyable by the players on my server.
I have added `claims.forceload.offlineForceload` to `[server]/config/openpartiesandclaims-server.toml` to the string arrays at `playerConfigurablePlayerConfigOptions` and `opConfigurablePlayerConfigOptions`.

I have also modified `offlineForceload = true` within `[server]/config/openpartiesandclaims-default-player-config.toml`.

Is there something im missing to allow players to forceload whilst offline?
